Documentation/persistentvolumeclaim-metrics.md

```diff
@@ -4,7 +4,7 @@
 | ---------- | ----------- | ----------- |
 | kube_persistentvolumeclaim_info | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; <br> `storageclass`=&lt;persistentvolumeclaim-storageclassname&gt; |
 | kube_persistentvolumeclaim_status_phase | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; <br> `phase`=&lt;Pending\|Bound\|Lost&gt; |
-| kube_persistentvolumeclaim_resource_requests_storage_bytes | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt; |
+| kube_persistentvolumeclaim_resource_requests_storage_bytes | Gauge | `namespace`=&lt;persistentvolumeclaim-namespace&gt; <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-name&gt;<br>`volumename`=&lt;volumename&gt; |
```

Comment:
As far as I can tell from your code changes, this is an additional label on the `kube_persistentvolumeclaim_info` metric, rather than the `kube_persistentvolumeclaim_resource_requests_storage_bytes` metric.

Reply:
Agree with @brancz. @svend the `volumename` label should be added to `kube-persistentvolumeclaim_info` instead of `kube_persistentvolumeclaim_resource_requests_storage_bytes`.

Others LGTM.